pkg/api/schema/schema.proto

```diff
@@ -464,6 +464,7 @@ message DatabaseSettings {
 message CreateDatabaseRequest {
 	string name = 1;
 	DatabaseNullableSettings settings = 2;
+	bool ifNotExists = 3;
 }
 
 message CreateDatabaseResponse {
```

Comment:
How about adding extra boolean to the response to indicate whether the database already existsed?